src/gleam_stdlib.mjs

```diff
@@ -73,6 +73,8 @@ export function string_length(string) {
       i++;
     }
     return i;
+  } else if (string === "") {
+    return 0;
```

Comment:
Can we put this as the first line of the function? No need to build the iterator etc if it's an empty string

Reply:
Good point. I can probably do that in ~36hrs or so, out of town atm.

Reply:
Updated.